pkg/ctxutil/ctxutil.go

```diff
@@ -15,12 +15,9 @@ const (
 	ctxKeyTerminal contextKey = iota
 	ctxKeyInteractive
 	ctxKeyStdin
-	ctxKeyNoPager
-	ctxKeyShowSafeContent
 	ctxKeyGitCommit
 	ctxKeyAlwaysYes
 	ctxKeyVerbose
-	ctxKeyNotifications
```

Comment:
I feel like most of the rest of `ctxutil` could be replaced with `SetEnv` configs, no?

Reply:
No. This is not possible (yet?).

So of these are used for in-process communication.
Maybe some day we can get rid of these. But it's so much work with so little user-visible value that I'm not very motivated to do it now.